eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/permission/PermissionGrantResult.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.permission;
+
+import java.util.List;
+
+
+/**
+ * Represents the result of the user's decision on a permission request.
+ *
+ * @author Gabor Keszthelyi
+ */
+public interface PermissionGrantResult
+{
+    /**
+     * The list of permissions granted by the user.
+     */
+    List<String> granted();
```

Comment:
Does it really make sense to return a list here? Wouldn't it be sufficient to have a single method like `boolean isGranted(String)`?